pbm/backup/logical.go

```diff
@@ -170,11 +170,6 @@ func (b *Backup) doLogical(
 	snapshotSize, err := snapshot.UploadDump(ctx,
 		dump,
 		func(ns, ext string, r io.Reader) error {
-			stg, err := util.StorageFromConfig(&b.config.Storage, l)
-			if err != nil {
-				return errors.Wrap(err, "get storage")
```

Comment:
do not remove this. it's made by intention. the S3 and Azure client' fields and their usage in methods are not synchronized yet to be used concurrently.

https://jira.percona.com/browse/PBM-944